src/sider/arbeidsuforhet/VurderingSkjema.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { Button, Heading, Textarea } from "@navikt/ds-react";
+import { ButtonRow } from "@/components/Layout";
+import { SkjemaInnsendingFeil } from "@/components/SkjemaInnsendingFeil";
+import { Forhandsvisning } from "@/components/Forhandsvisning";
+import { useForm } from "react-hook-form";
+import {
+  VurderingRequestDTO,
+  VurderingType,
+} from "@/data/arbeidsuforhet/arbeidsuforhetTypes";
+import { useArbeidsuforhetVurderingDocument } from "@/hooks/arbeidsuforhet/useArbeidsuforhetVurderingDocument";
+import { useSendVurderingArbeidsuforhet } from "@/data/arbeidsuforhet/useSendVurderingArbeidsuforhet";
+
+// TODO: Basere tittel og info på type
+const texts = {
+  title: (type: VurderingType) => {
+    return type === VurderingType.OPPFYLT
+      ? "Skriv en kort begrunnelse for hvorfor bruker oppfyller 8-4"
+      : "Skriv en kort begrunnelse for hvorfor bruker ikke oppfyller 8-4";
+  },
+  info: "Det du skriver her blir liggende i historikken på brukeren og et brev med begrunnelse blir sendt til bruker for deg.",
+  beskrivelseLabel: "Begrunnelse (obligatorisk)",
+  begrunnelseDescription: "Åpne forhåndsvisning for å se hele varselet.",
```

Comment:
Må også endre litt på denne siden det ikke er noe varsel. Men trenger vi egentlig noe forhåndsvisning av OPPFYLT / AVSLAG? Vi har ikke det for de vurderingene vi bare journalfører uten å sende til bruker i aktivitetskravet.
Dokumentet vi har lagt opp til å journalføre nå har ikke noe mer tekst enn bare navn på sykmeldt, type, begrunnelse og veileder.